src/components/CCPDashboard/PatientInfoTable.tsx

```diff
@@ -230,153 +270,200 @@ export const PatientInfoTable = ({
-            width="72px"
-            style={{ maxWidth: '72px' }}
-          >
-            {patient.gender}
-          </TableCell>
-          <TableCell
-            className={classes.tableCell}
-            width="34px"
-            style={{ maxWidth: '34px' }}
-          >
-            {patient.age}
-          </TableCell>
-          <TableCell
-            className={classes.tableCell}
-            width="104px"
-            style={{ maxWidth: '104px' }}
-          >
-            {statusLabels[patient.status]}
-          </TableCell>
-          {type === CCPDashboardTabOptions.PatientOverview && (
-            <>
-              <TableCell
-                className={classes.tableCell}
-                width="128px"
-                style={{ maxWidth: '128px' }}
-              >
-                {patient.hospitalId?.name}
-              </TableCell>
-              <TableCell
-                className={classes.tableCell}
-                width="192px"
-                style={{ maxWidth: '192px' }}
-              >
-                {moment(patient.updatedAt).format('MMM D YYYY, h:mm A')}
-              </TableCell>
-            </>
-          )}
-          {type === CCPDashboardTabOptions.Hospital && (
-            <>
-              <TableCell
-                className={classes.tableCell}
-                width="132px"
-                style={{ maxWidth: '132px' }}
-              >
-                {patient.runNumber}
-              </TableCell>
-              <TableCell
-                className={classes.tableCell}
-                width="192px"
-                style={{ maxWidth: '192px' }}
-              >
-                {moment(patient.transportTime).format('MMM D YYYY, h:mm A')}
-              </TableCell>
-            </>
-          )}
-          <TableCell width="48px" style={{ maxWidth: '48px' }}>
-            <IconButton
-              color="inherit"
-              onClick={(e) => handleClickOptions(e, patient)}
+            <TableCell
+              className={classes.tableCell}
+              style={{
+                width: '78px',
+                maxWidth: '78px',
+                borderLeft: `${isActive ? '16px' : '0px'} solid ${
+                  isActive
+                    ? triageLevels[patient.triageLevel].colour
+                    : Colours.InactiveGrey
+                }`,
+                color: `${isActive ? Colours.Black : Colours.InactiveGrey}`,
+              }}
             >
-              <MoreHoriz />
-            </IconButton>
-          </TableCell>
-        </TableRow>
-      );
-    }
-  );
+              {triageLevels[patient.triageLevel].label}
+            </TableCell>
+            <TableCell
+              className={classes.tableCell}
+              width="94px"
+              style={{
+                maxWidth: '94px',
+                color: `${isActive ? Colours.Black : Colours.InactiveGrey}`,
+              }}
+            >
+              {patient.barcodeValue}
+            </TableCell>
```

Comment:
can this be pulled out into a separate component/function? from what i can tell only the `width` and actual value are being changed and copy pasting it so many time hurts readability a lot - i suggest the below as an example
```suggestion
            ...
            const renderTableCell = (width, value) =>{
            <TableCell
              className={classes.tableCell}
              width={width}
              style={{
                maxWidth: {width},
                color: `${isActive ? Colours.Black : Colours.InactiveGrey}`,
              }}
            >
              {value}
            </TableCell>
            ...
            // maybe need to manipulate `patients` to add widths (or make another array in parallel with just the widths)
            ...
            patients.map(patient => renderTableCell(patient.width, patient.value))
```

Reply:
miscellaneous questions (maybe for @wongcmichelle as well?)
- how come we're managing the width of each cell in each individual cell? can't we set it somewhere else once and have it just inherit it?
- same deal with the text colour - very strange that it's being handled in each individual cell
- the border might mess with the setup i suggested above, might be worth taking the whole `style` object as a parameter

Reply:
@petorsong the fixed widths were determined by char - I worked with her on that when building this CCP dashboard initially